closeio/contrib/django/views.py

```diff
@@ -15,10 +15,35 @@
 
 
 class CloseIOWebHook(View):
+    # first elements are from webhook legacy API
```

Comment:
I strongly recommend to leave the existing webhook view as it is. We don't want to risk breaking existing functionality. And in general I think it's better to split concerns about the old and the new style webhooks instead of handling them inside the same view.

We could create a new view here that handles the new style webhooks, but that's currently not necessary. For this we need to think of an approach that makes sense because the new style webhooks work differently and we can provide different signals. This will be a topic when we migrate everything to new style webhooks, but for now, we don't need a change here.
For our current problem we'll simply add a view that handles the new webhook inside our application. There is no need to put this inside this library yet.